src/main/java/uk/ac/cam/cl/dtg/isaac/api/AssignmentFacade.java

```diff
@@ -1098,16 +1096,6 @@ public Response assignGameBoards(@Context final HttpServletRequest request,
                     this.userBadgeManager.updateBadge(currentlyLoggedInUser,
                             UserBadgeManager.Badge.TEACHER_ASSIGNMENTS_SET, assignmentWithID.getId().toString());
 
-                    tagsLoop:
```

Comment:
Not every day you see a labelled loop!

Reply:
No, I was surprised by this too! Didn't even know the syntax for it!